tensorflow/python/ops/linalg_grad.py

```diff
@@ -334,9 +335,10 @@ def _SvdGrad(op, grad_s, grad_u, grad_v):
     # multiple singular values with value zero. I am not sure if this is a true
     # instability or if it simply throws off the finite difference gradient
     # checker.
```

Comment:
How should I update this comment?  I personally feel there isn't much hope for the case `abs(m-n) > 1` when `full_matrices=True`, because the gradient would need to be consistent with the particular choices of U and V made by the SVD algorithm.  So my preference is to remove the comment.  But if others prefer for me to leave it here, I can instead change `numerically stable for abs(m-n) > 1` to `numerically stable for abs(m-n) > 1 and full_matrices=True`.

Reply:
I'm OK with removing the comment.

Reply:
I'm OK with removing the comment.